inc/apiclient.class.php

```diff
@@ -241,9 +257,15 @@ public function createForeignKey($f_id_src = 0, $f_id_trgt = 0)
             return false;
         }
 
-        $data = $this->httpQuery("/api/field/$f_id_src", [
+        $mb_version = $this->getVersion();
+
+        $mb_fieldname = version_compare($mb_version['tag'], 'v0.39.0', '>=')
+            ? 'semantic_type'
+            : 'special_type';
+
+        $data = $this->httpQuery("field/$f_id_src", [
             'json' => [
-                'special_type'       => 'type/FK',
+                $mb_fieldname        => 'type/FK',
```

Comment:
Doesn't it work with both?
```suggestion
                'special_type'        => 'type/FK', // version < 0.39
                'semantic_type'       => 'type/FK', // version >= 0.39
```